manual_jb/content/account.md

```diff
@@ -1,15 +1,15 @@
 # IN-CORE Account
 
-Access to the **IN-CORE** service requires a user account provided and operated by the [NCSA](http://www.ncsa.illinois.edu/) identity management system.
-Registering for an account is free and open to everyone. This account gives you access to all of the public data on
-the system and allows you to create data that is only accessible by you. If applicable, it is recommended
-that you sign up with your institutional email. To register for an account, please go to:
+Access to the **IN-CORE** service requires a user account. Registering for an account is free and open to everyone.
+This account gives you access to all of the public data on the system and allows you to create data that is only
+accessible by you. If applicable, it is recommended that you sign up with your institutional email.
+To register for an account, please go to:
 
-[https://identity.ncsa.illinois.edu/register/BSKC2UKQPU](https://identity.ncsa.illinois.edu/register/BSKC2UKQPU)
+[https://incore.ncsa.illinois.edu/auth/realms/In-core/login-actions/registration?
+client_id=react-auth](https://incore.ncsa.illinois.edu/auth/realms/In-core/login-actions/registration?client_id=react-auth)
```

Comment:
You might need to update this to another pattern:
https://incore.ncsa.illinois.edu/auth/realms/In-core/protocol/openid-connect/registrations?client_id=react-auth&response_type=code&scope=openid&redirect_uri=https://incore.ncsa.illinois.edu/

Reply:
Good catch! I will update this shortly.